src/Signature/SignatureV4.php

```diff
@@ -326,7 +358,8 @@ private function moveHeadersToQuery(array $parsedRequest)
             if (substr($lname, 0, 5) == 'x-amz') {
                 $parsedRequest['query'][$name] = $header;
             }
-            if ($lname !== 'host') {
+            $blacklist = $this->getHeaderBlacklist();
+            if (isset($blacklist[$lname]) || $lname === strtolower(self::AMZ_CONTENT_SHA256_HEADER)) {
```

Comment:
Can you please update the formatting for this line, since it's over 80 characters?

```php
if (isset($blacklist[$lname])
    || $lname === strtolower(self::AMZ_CONTENT_SHA256_HEADER)
) {
    unset($parsedRequest['headers'][$name]);
}
```